layout.yaml

```diff
@@ -2,37 +2,37 @@ teams:
   - name: level-2-bar
     display_name: Level 2 Bar
     teams:
-      - ABS
-      - BPV
-      - BRK
-      - CAG
+      - QMC
```

Comment:
I can't remember exactly, but I don't think the ordering here has any meaning for SRComp? I'd be tempted to keep these alphabetical per section, unless something else is driving the ordering?

Reply:
I had ordered them roughly by distance to the arena within each zone with the vague recollection teams closer to the arena are in earlier matches. But if it doesn't work how I'm thinking then happy for it to be alphabetical :)

Reply:
Do check the code, but I think that SRComp treats this just as a collection rather than anything more.
There is support for different warning timings per shepherding zone I think, but not within a zone.

Reply:
I think [order_teams_by_location](https://github.com/PeterJCLaw/srcomp-cli/blob/d3dae2a5ce8e36fd5f86e9b57fcf1709e9e008b9/sr/comp/cli/import_schedule/teams_mapping.py#L30) will mean that when importing the league teams closest to the arena will be scheduled first

Reply:
Ah, I'd forgotten that this file was how that worked :+1: